vendor-bundles.js

```diff
@@ -0,0 +1,38 @@
+// Defines a set of vendor bundles which are
+// libraries of 3rd-party code referenced by
+// one or more bundles of Hypothesis client/frontend
+// code
+
+/**
+ * Decorate a module ID with a 'doNotParse' property
+ * indicating that the module bundler should not try
+ * to parse it for require() statements.
+ */
+function doNotParse(id) {
```

Comment:
TODO: `module.exports` currently has this type:

```ts
interface BundleMap {
  [index: string]: Array<string | { name: string; noParse: boolean }>
}
```

It would be easier to understand if this was just `[index: string]: Array<{name: string, noParse?: boolean}>` instead.